simulator/src/main/java/com/hedera/block/simulator/generator/BlockAsFileLargeDataSets.java

```diff
@@ -65,6 +65,14 @@ public BlockAsFileLargeDataSets(@NonNull BlockGeneratorConfig config) {
         this.formatString = "%0" + config.paddedLength() + "d" + config.fileExtension();
     }
 
+    /**
+     * Initialize the block stream manager and load blocks into memory.
+     */
+    @Override
+    public void init() {
+        // Do nothing, because we don't have real initializing and loading blocks into memory for this implementation.
+    }
```

Comment:
You should be able to remove this `init()` method. See my comment in `BlockStreamManager`

Reply:
Fixed 👍